src/actions/guides/frontend/rendering_html.cr

```diff
@@ -962,6 +962,50 @@ class Guides::Frontend::RenderingHtml < GuideAction
     ```erb
     My name is: <%= @user.name %>
     ```
+
+    #{permalink(ANCHOR_RENDERING_TEMPLATES)}
+    ## Caching HTML Content
+
+    Lucky applications can leverage [lucky_cache](https://github.com/luckyframework/lucky_cache) to cache HTML content within pages. By caching parts of your application, Lucky can serve already rendered and processed pages resulting in faster page loads for you application.
+
+    To begin utilizing `lucky_cache`, add it as a dependency to your `shards.yml` file.
+    ```
+    dependencies:
+      lucky_cache:
+        github: luckyframework/lucky_cache
+    ```
+
+    Then run `shards install` to update your dependencies.
+
+    Next, you'll need to require `lucky_cache` in your application in `src/shards.cr`. Lastly, create a config file in `config/lucky_cache.cr` where you will configure the cache options.
+    ```
+    require "lucky_cache"
```

Comment:
```suggestion
    # config/lucky_cache.cr
    require "lucky_cache"
```
In code examples like this, I like to add a comment at the top with the path. It makes it easier to see where you're adding that bit of code